wiki/markdown/usage/service.md

```diff
@@ -122,10 +122,10 @@ Status codes are generated according to the below table.
 | 404         | Not Found             | Requested template is not available               | The endpoint is not found, or template for specified event type is not found                                            |
 | 406         | Not Acceptable        | No event id found with ERLookup properties        | Is returned if no event id fetched from configured event repository in REMReM generate.                                 |
 | 417         | Expectation Failed    | Multiple event ids found with ERLookup properties | Is returned if multiple event ids fetched from configured event repository in REMReM generate.                          |
+| 422         | Unprocessable Entity  | Link specific options could not fetch information from ER   | Is returned if could not fetch information from ER with specific options.                                     |
```

Comment:
See my earlier comment about the error message

Reply:
changed error message as suggested

Reply:
Thanks